src/main/java/frc/robot/subsystems/ShooterPivot.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class ShooterPivot extends SubsystemBase {
+  private CANSparkMax motor;
+  private CANcoder cancoder;
+  private double m_setPoint;
+  private PIDController pid;
+  private ArmFeedforward ff;
+  /** Creates a new ShooterPiviot. */
+  public ShooterPivot() {
+    motor = new CANSparkMax(1, MotorType.kBrushless);
+    cancoder = new CANcoder(0);
+    m_setPoint = 0;
+    pid = new PIDController(0, 0, 0);
+    this.ff = new ArmFeedforward(0, 0, 0, 0);
+
+  }
+
+  private double encoderAngle() {
+    StatusSignal<Double> position = cancoder.getPosition();
+    Double positionValue = position.getValue();
+    return positionValue.doubleValue();
+  }
+
+  public void moveShooterPivot(double setPoint) {
+    m_setPoint = setPoint;
+  }
+
+  private double encoderPosition() {
```

Comment:
What is the difference between this method and `encoderAngle()`. I know you are using a different method from `CANCoder`, but what is the difference between them? Is there a reason we use both? Should we only use one?